### ISSUE TITLE
[Bug]: <Unable to upgrade CLI on windows>

### ISSUE THREAD

**Issue by iFAXec** [Shopify/cli#2675]
### Please confirm that you have:

- [X] Searched [existing issues](.) to see if your issue is a duplicate. (If you’ve found a duplicate issue, feel free to add additional information in a comment on it.)
- [X] Reproduced the issue in [the latest CLI version](https://www.npmjs.com/package/@shopify/cli).

### In which of these areas are you experiencing a problem?

Theme

### Expected behavior

The CLI should have been upgrade to CLI3.X however, when I input my upgrade comman, I get an error the "Missing script: 'shopify'"

I have quoted steps that I followed in the CLI below 

`→ C:\Users\User› shopify version
* A new version of Shopify CLI is available! You have version 2.16.1 and the latest version is 2.36.0.

  To upgrade, follow the instructions for the package manager you’re using:
  https://shopify.dev/tools/cli/troubleshooting#upgrade-shopify-cli

2.16.1
→ C:\Users\User› npm run shopify upgrade
npm ERR! Missing script: "shopify"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in: C:\Users\User\AppData\Local\npm-cache\_logs\2023-08-20T10_14_41_363Z-debug-0.log
→ C:\Users\User› npm install -g @shopify/cli @shopify/theme

changed 887 packages in 41s

104 packages are looking for funding
  run `npm fund` for details
→ C:\Users\User› shopify version
* A new version of Shopify CLI is available! You have version 2.16.1 and the latest version is 2.36.0.

  To upgrade, follow the instructions for the package manager you’re using:
  https://shopify.dev/tools/cli/troubleshooting#upgrade-shopify-cli

2.16.1
→ C:\Users\User› npm run shopify upgrade
npm ERR! Missing script: "shopify"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in: C:\Users\User\AppData\Local\npm-cache\_logs\2023-08-20T10_17_22_279Z-debug-0.log
→ C:\Users\User›`

### Actual behavior

The CLI should be upgraded to 3.X as these same commands/steps to upgrade works on my friends laptop.

### Reproduction steps

`→ C:\Users\User› shopify version
* A new version of Shopify CLI is available! You have version 2.16.1 and the latest version is 2.36.0.

  To upgrade, follow the instructions for the package manager you’re using:
  https://shopify.dev/tools/cli/troubleshooting#upgrade-shopify-cli

2.16.1
→ C:\Users\User› npm run shopify upgrade
npm ERR! Missing script: "shopify"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in: C:\Users\User\AppData\Local\npm-cache\_logs\2023-08-20T10_14_41_363Z-debug-0.log
→ C:\Users\User› npm install -g @shopify/cli @shopify/theme

changed 887 packages in 41s

104 packages are looking for funding
  run `npm fund` for details
→ C:\Users\User› shopify version
* A new version of Shopify CLI is available! You have version 2.16.1 and the latest version is 2.36.0.

  To upgrade, follow the instructions for the package manager you’re using:
  https://shopify.dev/tools/cli/troubleshooting#upgrade-shopify-cli

2.16.1
→ C:\Users\User› npm run shopify upgrade
npm ERR! Missing script: "shopify"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in: C:\Users\User\AppData\Local\npm-cache\_logs\2023-08-20T10_17_22_279Z-debug-0.log
→ C:\Users\User›`

### Operating System

Windows10

### Shopify CLI version (check your project's `package.json` if you're not sure)

2.16.1

### Shell

Powershell

### Node version (run `node -v` if you're not sure)

18.17.1

### What language and version are you using in your application?

Node 18.17.1

**Comment by isaacroldan** [comment]
Hi! I think there is a bit of confusion between cli versions and how to upgrade.

- Your original CLI installation is version `2.x` (which is deprecated)
- The upgrade docs you link are for `apps`, not for `themes`. [This is the one for themes](https://shopify.dev/docs/themes/tools/cli#upgrade-shopify-cli)
- But what you actually need is to `migrate` to CLI 3.x (not just upgrade)
- [These](https://shopify.dev/docs/themes/tools/cli/migrate) are the docs you should follow to migrate to CLI 3.x

Basically you run `npm install -g @shopify/cli @shopify/theme` to install CLI 3.x for themes.
Beware of having CLI 2.x and 3.x installed at the same time, they might conflict, you should uninstall CLI 2.x first.

Hope this helps, i'm going to close this issue as this is not a bug on the CLI. But feel free to ask any question about migration process.

**Comment by iFAXec** [comment]
Thank you very much @isaacroldan. It worked.
